source/runner/_tests/_executor.test.ts

```diff
@@ -172,10 +174,7 @@ describe("setup", () => {
     const platform = new FakePlatform()
     const exec = new Executor(new FakeCI({}), platform, inlineRunner, defaultConfig, new FakeProcces())
     const dsl = await defaultDsl(platform)
-    const apiFailureMock = jest.fn().mockReturnValue(
-      new Promise<any>((_, reject) => reject())
-    )
-    platform.createInlineComment = apiFailureMock
+    platform.createInlineComment = jest.fn().mockReturnValue(new Promise<any>((_, reject) => reject()))
```

Comment:
not really a change.
prettier reformatted these lines, and I removed the redundant `apiFailureMock`.